packages/components/src/Button.tsx

```diff
@@ -74,70 +92,82 @@ const BaseButton = styled(ButtonComponent)`
+    primaryButtonDisabledColor: colors.basic.WHITE,
+  },
+}
+
 export const InvertedButton = BaseButton.extend`
   text-transform: uppercase;
-  background-color: ${colors.basic.WHITE};
-  color: ${colors.accent.CIVIL_BLUE};
-  border: 2px solid ${colors.accent.CIVIL_BLUE};
+  background-color: ${props => props.theme.invertedButtonBackground};
+  color: ${props => props.theme.invertedButtonColor};
+  border: 2px solid ${props => props.theme.invertedButtonColor};
   &:hover {
-    background-color: ${colors.accent.CIVIL_BLUE};
-    color: ${colors.basic.WHITE};
+    background-color: ${props => props.theme.invertedButtonColor};
+    color: ${props => props.theme.invertedButtonBackground};
   }
 `;
 
-export const SecondaryButton = BaseButton.extend`
-  background-color: ${colors.basic.WHITE};
-  color: ${colors.accent.CIVIL_GRAY_2};
-  border: 1px solid ${colors.accent.CIVIL_GRAY_3};
-  &:hover {
-    background-color: ${colors.accent.CIVIL_BLUE};
-    border: 1px solid ${colors.accent.CIVIL_BLUE};
-    color: ${colors.basic.WHITE};
+InvertedButton.defaultProps = {
+  theme: {
+    invertedButtonBackground: colors.basic.WHITE,
+    invertedButtonColor: colors.accent.CIVIL_BLUE,
   }
-  &.active {
-    background-color: ${colors.accent.CIVIL_BLUE};
-    border: 1px solid ${colors.accent.CIVIL_BLUE};
-    color: ${colors.basic.WHITE};
+}
+
+export const SecondaryButton = BaseButton.extend`
+  background-color: ${props => props.theme.secondaryButtonBackground};
+  color: ${props => props.theme.secondaryButtonColor};
+  border: 1px solid ${props => props.theme.secondaryButtonBorder};
+  &:hover, &.active {
+    background-color: ${props => props.theme.secondaryButtonHoverBackground};
+    border: 1px solid ${props => props.theme.secondaryButtonHoverBackground};
+    color: ${props => props.theme.secondaryButtonHoverColor};
   }
 `;
 
+SecondaryButton.defaultProps = {
+  theme: {
+    secondaryButtonBackground: colors.basic.WHITE,
+    secondaryButtonColor: colors.accent.CIVIL_GRAY_2,
+    secondaryButtonBorder: colors.accent.CIVIL_GRAY_3,
+    secondaryButtonHoverBackground: colors.accent.CIVIL_BLUE,
+    secondaryButtonHoverColor: colors.basic.WHITE,
+  },
+}
+
 export const DarkButton = BaseButton.extend`
-  background-color: ${colors.primary.BLACK};
-  color: ${colors.basic.WHITE};
-  border: 1px solid ${colors.primary.BLACK};
-  &:hover {
-    background-color: ${colors.accent.CIVIL_GRAY_1};
-    border: 1px solid ${colors.accent.CIVIL_GRAY_1};
-    color: ${colors.basic.WHITE};
-  }
-  &.active {
-    background-color: ${colors.accent.CIVIL_GRAY_1};
-    border: 1px solid ${colors.accent.CIVIL_GRAY_1};
-    color: ${colors.basic.WHITE};
+  background-color: ${props => props.theme.darkButtonBackground};
+  color: ${props => props.theme.darkButtonColor};
+  &:hover, &.active {
+    background-color: ${props => props.theme.darkButtonHoverBackground};
   }
 `;
 
-export const CancelButton = SecondaryButton.extend`
```

Comment:
This looks cool to me. 

My only comment is that I am using `CancelButton` in the DApp now, because it was defined and available.

I don't necessarily need to use it, but we should have a `TextButton` for cases where we have a button but the style looks like plain text or a link.

Reply:
I only deleted it because i searched and no one was using it